.travis.yml

```diff
@@ -11,18 +11,42 @@ cache:
     - "bower_components"
 
 script:
+  - zip -r latest *
```

Comment:
This ends up adding all of `node_modules` as well. I'm not certain how `codedeploy` works. Is it going to run `npm install` on the server anyway? If so, we should remove node_modules from here.